tests/UnleashTest.php

```diff
@@ -6,7 +6,7 @@
 use GuzzleHttp\Handler\MockHandler;
 use GuzzleHttp\HandlerStack;
 use GuzzleHttp\Psr7\Response;
-use PHPUnit\Framework\Assert;
+use PHPUnit_Framework_Assert as Assert;
 use PHPUnit\Framework\TestCase;
```

Comment:
Does this need to change?

Reply:
surprisingly no -- not sure why some classes use proper namespacing...